.github/workflows/ci.yml

```diff
@@ -42,24 +42,24 @@ jobs:
         COVERALLS_TOKEN: ${{ secrets.github_token }}
       run: goveralls -coverprofile=covprofile -service=github -ignore="confgenerator/filter/internal/generated/*/*"
     - id: 'set-coverage-vars'
-      if: ${{ (runner.os == 'Linux') && github.event.pull_request.merged && (github.event.pull_request.head.repo.full_name == 'GoogleCloudPlatform/ops-agent') }}
+      if: ${{ runner.os == 'Linux' && github.event_name == 'push' && github.ref == 'refs/heads/master' }}
```

Comment:
You probably still want to check `github.repository`, because the forks would also have a `master` branch…